src/templates/article.jsx

```diff
@@ -1,40 +1,106 @@
+  avatarContainer: {
+    display: "flex",
+    "& > *": {
+      marginRight: theme.spacing(1),
+    },
+  },
+  link: {
+    color: theme.palette.text.secondary,
+    verticalAlign: "center",
+  },
+  chips: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
 const Article = ({ data, pageContext }) => {
-  const { frontmatter, body } = data.mdx;
-  console.log(pageContext);
+  const { frontmatter, body, timeToRead } = data.mdx;
+  const classes = useStyles();
+  const related = [];
+  if(pageContext.prev) related.push(pageContext.prev);
+  if(pageContext.next) related.push(pageContext.next);
   return (
     <Layout>
       <SEO 
         title={frontmatter.title} 
         description={body.substring(0,140).replace(/^#.*?$/gm, "")} 
         article={true} 
       />
-      <Typography variant="h1">{frontmatter.title}</Typography>
-      <Typography variant="subtitle1">{frontmatter.date}</Typography>
+      {/*Title*/}
+      <Typography variant="h3">{frontmatter.title}</Typography>
+      {/*Authors*/}
+      <Grid container alignItems="center">
+        {/*Author Pictures*/}
+        <Grid item>
+          <div className={classes.avatarContainer}>
+          {frontmatter.authors.map(a => 
+            <Avatar alt={a.display_name} src={a.image} title={a.display_name}>{a.username.substring(0,1)}</Avatar>
+          )}
+          </div>
+        </Grid>
+        {/*Author Names/Links*/}
+        <Grid item>
+          <Typography variant="subtitle1">
+            By {frontmatter.authors.map((a, index, array) => 
+              <Fragment>
+                <Link component={GatsbyLink} to={`/author/${a.username}`} className={classes.link}>
+                  {a.display_name}
+                </Link>
+                {index === array.length-2 && " and "}
+                {index < array.length-2 && ", "}
+              </Fragment>
+            )}
+          </Typography>
+        </Grid>
+      </Grid>
+      {/*Date Published*/}
+      <Typography variant="subtitle1" component="time">
+        {" " + frontmatter.date + " "}&middot;{" " + timeToRead + " min read"}
+      </Typography>
       <MDXRenderer>{body}</MDXRenderer>
-      {pageContext.prev && (
-        <Link to={`/articles/${pageContext.prev.node.fields.slug}`}>
-          <Fragment>
-            {"<--"}
-            {pageContext.prev.node.frontmatter.title}
-          </Fragment>
-        </Link>
-      )}
-      {pageContext.next && (
-        <Link to={`/articles/${pageContext.next.node.fields.slug}`}>
-          <Fragment>
-            {"-->"}
-            {pageContext.next.node.frontmatter.title}
-          </Fragment>
-        </Link>
-      )}
+      <p>
+        Tagged: {frontmatter.tags.map(tag => 
+          <TagChip className={classes.chips} name={tag} label={tag} />
+        )}
+      </p>
+      <Grid container justify="space-between">
+        {pageContext.prev && (
+          <Grid item>
+            <Grid container alignItems="center">
+              <ArrowBackIcon />
```

Comment:
I think users would assume that the arrow is part of the link. Although when I tried putting it in the link, it messed up the styling. Do you know any way to fix this?

Reply:
I tried to align the link text baseline and arrow to no avail, so I left the arrow out of the link. My solution is to replace these left/right links with an ArticleList component titled "From the Issue". The ArticleList component is used in tons of other places on the site, so it would be familiar to readers. Unfortunately, the pageContext queries that @kkhan01 uses in `gatsby-node.js` don't follow the shape expected in ArticleList (namely, they're missing a description). 

I'm not sure how to fix that, so I left the left/right links in place.

Reply:
I guess this is fine to leave as is for now. Maybe we can look back at it later to fix the CSS.